.travis.yml

```diff
@@ -8,68 +8,47 @@ php:
 
 env:
   - DRUPAL_CORE=8.1.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
   - DRUPAL_CORE=8.2.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
+  - REVIEW_STANDARD="Drupal"
+    CODE_REVIEW=1
+  - REVIEW_STANDARD="DrupalPractice"
+    CODE_REVIEW=1
```

Comment:
I'd like to join this one. In one of our volunteering project I added code sniffer. I tackle this in another way that I think will be better for this one - https://github.com/hasadna/open_pension/pull/131/files#diff-354f30a63fb0907d4ad57269548329e3R14.